server/src/test/java/io/spine/server/command/model/CommandHandlerMethodTest.java

```diff
@@ -216,6 +216,9 @@ void messageParam() {
         }
 
         @Test
+        @MuteLogging
```

Comment:
We rarely use single-line comments.

I would rewrite this one (and the one above) like that:

```java
@MuteLogging  /** Signature mismatch warnings are expected. */
```